src/main/java/io/github/railroad/project/ui/welcome/WelcomeLeftPane.java

```diff
@@ -29,33 +26,40 @@ public WelcomeLeftPane() {
 
         var hbox = new HBox();
         hbox.setAlignment(Pos.CENTER);
-        hbox.getChildren().add(new ImageView(new Image(Railroad.getResourceAsStream("images/logo.png"), 100, 100, true, true)));
+        hbox.getChildren().add(new ImageView(
+                new Image(Railroad.getResourceAsStream("images/logo.png"), 100, 100, true, true)));
 
         var rightVbox = new VBox();
         rightVbox.setAlignment(Pos.CENTER);
         rightVbox.getChildren().add(new Label("Railroad IDE"));
-        rightVbox.getChildren().add(new Label("1.0.0"));
+        rightVbox.getChildren().add(new Label("1.0.0(dev)"));
 
         hbox.getChildren().add(rightVbox);
         topBox.getChildren().add(hbox);
 
         listView.getItems().addAll(MenuType.values());
-        listView.getSelectionModel().select(MenuType.HOME);
         listView.setCellFactory(param -> new MenuTypeCell());
 
         setPrefWidth(200);
         setAlignment(Pos.TOP_CENTER);
-        getChildren().addAll(topBox, new Separator(), listView);
         VBox.setVgrow(listView, Priority.ALWAYS);
+
+        getChildren().addAll(
+                topBox,
+                new Separator(),
+                listView
+        );
     }
 
     public ListView<MenuType> getListView() {
         return listView;
     }
 
     public enum MenuType {
-        HOME("Home", FontAwesomeSolid.HOME, Color.RED),
-        SETTINGS("Settings", FontAwesomeSolid.COG, Color.BLUE);
+        NEW_PROJ("New Project", FontAwesomeSolid.PLUS, Color.BLACK),
+        OPEN_PROJ("Open Project", FontAwesomeSolid.FOLDER_OPEN, Color.BLACK),
+        IMPORT_PROJ("Import Project", FontAwesomeSolid.FILE_IMPORT, Color.BLACK),
```

Comment:
I'm not a fan of these being shortened to `PROJ` tbh, it just seems a bit silly.